### PR TITLE
Update zammad_ssl.conf

### PR DESCRIPTION
**Pull request by Impyus**
Proposed change to the NGINX SSL Conf template to correct duplicate server block and correct syntax errors.

### CODE REVIEW

**Comment by thorsteneckel** [comment]
Thanks for your proposal! That looks much better now. Can you please change the indentation because it's very hard to reason what the actual changes were if each line is marked as "changed".

**Comment by Impyus** [comment]
Sure, can you clarify what you mean by change the indentation?

**Comment by thorsteneckel** [comment]
Sure. Let's take [this line](https://github.com/zammad/zammad/pull/2072/files#diff-33ddb85c4e705cd0b5993e64efa5e3bcR12) for example. The only difference is that you added two spaces in front (indentation). But the general functionality stays the same. The diff shows that it was changed but actually wasn't. Just added whitespaces => indentation. So please remove those so we can see what actually changed. Thanks!

**Comment by Impyus** [comment]
Ah yes I understand you now, I think I've sorted that.

File: contrib/nginx/zammad_ssl.conf [hunk 1]
```diff
@@ -17,35 +18,33 @@ upstream zammad-websocket {
...
-  ssl_certificate /etc/nginx/ssl/example.com-fullchain.pem;
-  ssl_certificate_key /etc/nginx/ssl/example.com-privkey.pem;
+  ssl_certificate /etc/nginx/ssl/your_cert.crt;
+  ssl_certificate_key /etc/nginx/ssl/your_private_key.key;
 
-  ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+  ssl_protocols TLSv1.2;
 
-  ssl_ciphers HIGH:!aNULL:!MD5;
+  ssl_ciphers 'EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH';
```
Comment (monotek) [inline contrib/nginx/zammad_ssl.conf:47]: Did you checked that these ciphers are available on all supported platforms?

File: contrib/nginx/zammad_ssl.conf [hunk 2]
```diff
@@ -57,84 +56,46 @@ server {
   ssl_stapling on;
   ssl_stapling_verify on;
 
-  ssl_trusted_certificate /etc/nginx/ssl/lets-encrypt-x3-cross-signed.pem;
+  ssl_trusted_certificate /etc/nginx/ssl/your_cert.crt;
```
Comment (monotek) [inline contrib/nginx/zammad_ssl.conf:59]: This conflicts with the command in line 7.

**Comment by thorsteneckel** [comment]
@monotek - what do you think?

**Comment by thorsteneckel** [comment]
@monotek - ping

**Comment by monotek** [comment]
My remarks above still seem to apply...

File: contrib/nginx/zammad_ssl.conf (hunk 2 above)
Comment (Impyus) [reply contrib/nginx/zammad_ssl.conf:59]: Fair point, but perhaps the guidance @ line 7 should be changed to reflect. I renamed it so it was obvious that this was referring to your certificate whether you were using LetsEncrypt or not and would need to be changed to reflect it. I'm happy to add a readme to provide clarification on editing each section of the file for the end user.

File: contrib/nginx/zammad_ssl.conf (hunk 1 above)
Comment (Impyus) [reply contrib/nginx/zammad_ssl.conf:47]: As far as I understand it, these ciphers are supported on current releases of openSSL and by Apache, Nginx and Lighttpd. The reasoning for change here is that TLS 1.0 and 1.1 are deprecated. TLS 1.2 is the standard and this set-up enables the most secure config of the webserver. Again alternatives can be offered in the readme I'll create if users wish to enable weak ciphers for some reason. As it currently stands this config achieves an A+ rating on SSL Labs.

**Pull request merged**